### PR TITLE
bib-tool 2.65

### PR DESCRIPTION
**Pull request by J12D**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/bib-tool.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class BibTool < Formula
   desc "Manipulates BibTeX databases"
   homepage "http://www.gerd-neugebauer.de/software/TeX/BibTool/en/"
-  url "https://github.com/ge-ne/bibtool/archive/BibTool_2_64.tar.gz"
-  sha256 "946b218f04f654e74712eba300a4e21dc62247a92a39b02d85b90dce97a22ff4"
+  url "http://mirrors.ctan.org/biblio/bibtex/utils/bibtool/BibTool-2.65.tar.gz"
```
Comment (DomT4) [inline Formula/bib-tool.rb]: I'd rather not move away from the `https://` GH link unless there's a good reason to?

**Comment by J12D** [comment]
@DomT4 I changed the url, thought it'd be better to link to the ctan repo but you have a fair point. Do I need to squash my commits?

**Comment by DomT4** [comment]
@J12D Yeah, we prefer https as much as possible for security/privacy. Squashing would be perfect, if you could that'd be great.

**Comment by J12D** [comment]
Squashed commits in #2201

**Pull request closed without merging**